mock-backend.js

```diff
@@ -8,33 +8,36 @@ function genUID() {
 
 // Adds a restaurant and returns the restaurant ID
 function createRestaurant(name, latitude, longitude) {
-    restoID = 'resto_' + genUID()
-    restaurants[restoID] = {
+  var res_id = 'resto_' + genUID()
+    restaurants[res_id] = {
         name,
         location: {
             latitude,
             longitude
         },
-        menu: []
+        dishes: []
```

Comment:
I'd probably leave this as 'menu' here. At this level, we don't need to be so concerned about matching the terminology of the Zomato API, especially since the way the most people refer to a list of dishes at a restaurant is with the word "menu", not dishes.

Also, a good practice when committing code like this is to make a separate PR for each code change; in other words, make a PR for changing the variable names, and then another PR for logic additions/changes, etc. It makes it a lot easier to reason about what the code changes are and what they might potentially affect.

Reply:
sure